tools/bin/raiderStats.py

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env python3
+#!/data2/ssangha/conda_installation/stable_jan10_2020/ariatools_conda/envs/RAiDER/bin/python3
```

Comment:
Looks like you made a temporary change, just need to switch back.

Reply:
Ok, fixed.